lib/manager/gradle/index.ts

```diff
@@ -156,4 +161,13 @@ async function getGradleCommandLine(
   return cmd + ' ' + GRADLE_DEPENDENCY_REPORT_OPTIONS;
 }
 
+async function canExecute(path) {
```

Comment:
```suggestion
async function canExecute(path: string): boolean {
```

Reply:
Hmm, given that the function is not exported, I'm not sure why we can't let the compiler infer the types, but this is fine too.